Tasks/Common/webdeployment-common/deployusingmsdeploy.ts

```diff
@@ -65,4 +72,25 @@ export async function DeployUsingMSDeploy(webDeployPkg, webAppName, publishingPr
             tl.rmRF(setParametersFile, true);
         }
     }
-}
\ No newline at end of file
+}
+
+export async function executeMSDeploy(msDeployCmdArgs, retryCount, retryInterval, errObj) {
+    var deferred = Q.defer();
+    try {
+        await tl.exec("msdeploy", msDeployCmdArgs, <any>{failOnStdErr: true, errStream: errObj});
+        deferred.resolve("Azure App service successfully deployed");
+    } catch (error) {
```

Comment:
Question - Is this specific to deployment using msdeploy? Are we not seeing this for Kudu based deployments?